ui/desktop/src/config.ts

```diff
@@ -30,6 +30,56 @@ export type SystemConfig =
       name: string;
     };
 
+// Local storage key for extension configs
+const EXTENSION_CONFIGS_KEY = 'goose_extension_configs';
```

Comment:
Is this related to 

const saved = localStorage.getItem("user_settings");

at all? can't quite follow the relationship (it also has models and extensions)

Reply:
are there some old settings screens around?

Reply:
Oh good call - I didn't realize we already had settings stored this way